cashews/wrapper/wrapper.py

```diff
@@ -81,6 +82,7 @@ def _add_backend(self, backend: Backend, middlewares=(), prefix: str = default_p
             backend,
             middlewares + tuple(self._default_middlewares),
         )
+        self._sorted_prefixes = tuple(sorted(self._backends.keys(), reverse=True))
```

Comment:
@Krukov In the current context, it is not clear why you are converting a list to a tuple. You might want to check if you really need to do this.